### PR TITLE
Improve OCSP testing coverage

### PR DESCRIPTION
**Pull request by vincentkam**
- Add ECDSA certificate chain + scripts
 - Add scripts for testing RSA certs with delegate responders
 - Update RSA server certs to use multiple responders

### CODE REVIEW

**Comment by vincentkam** [comment]
Ready for review!

File: .evergreen/ocsp/certs.yml [hunk 1]
```diff
@@ -61,8 +61,49 @@ certs:
...
       IP: 127.0.0.1
-    authorityInfoAccess: 'OCSP;URI:http://localhost:8100/status/'
+    authorityInfoAccess: 'OCSP;URI:http://localhost:9001/power/level/,OCSP;URI:http://localhost:8100/status/'
     mustStaple: true
     subjectKeyIdentifier: hash
     keyUsage: [digitalSignature, keyEncipherment]
     extendedKeyUsage: [serverAuth, clientAuth]
+
+- name: 'ocsp_responder.crt'
+  description: Certificate and key for the OCSP responder
```
Comment (vincentkam) [inline .evergreen/ocsp/certs.yml]: We previously added the ocsp_responder certs but neglected to update the yml file

File: .evergreen/ocsp/README.md [hunk 2]
```diff
@@ -2,10 +2,12 @@
 
 The test certificates here were generating using the server team's [`mkcert.py`](https://github.com/mongodb/mongo/blob/master/jstests/ssl/x509/mkcert.py) tool.
 
-In order to generate a fresh set of certificates, clone the [`mongo` repository](https://github.com/mongodb/mongo) and run the following command from the root of the `mongo` repository:
+In order to generate a fresh set of certificates, clone this branch on my fork of the [`mongo` repository](https://github.com/vincentkam/mongo/tree/mkcert-ecdsa) and run the following command from the root of the `mongo` repository:
```
Comment (jmikola) [inline .evergreen/ocsp/README.md]: Is this a temporary change? For style reasons, I think we should avoid using personalized language (e.g. "my fork") in documents.

Comment (vincentkam) [reply .evergreen/ocsp/README.md]: Ah okay, I've updated the language to say "a fork" until my PR to the mongo repo gets approved

**Pull request merged**